www/news.php

```diff
@@ -113,7 +113,6 @@ function queryNewsRss(&$items, $db, $sourceType, $sourceID, $maxItems,
         $nuname = htmlspecialcharx($nuname);
         $nunameOrig = htmlspecialcharx($nunameOrig);
         $rssdate = date("D, j M Y H:i:s ", strtotime($ncre)) . 'UT';
-        list($rbody, $len, $trunc) = summarizeHtml($body, 210);
```

Comment:
I'm pretty sure the idea was to use the shortened description in the RSS.

You're deciding it's OK to return everything?

Reply:
I hate shortened descriptions in RSS! Eventually I want to unshorten all descriptions in all RSS feeds.

For example, when there's a new game listing, it should include something as close as possible to the game listing page, including a play online button, full description, game details, external links.